tests/sb-util.test.ts

```diff
@@ -1,226 +1,244 @@
-import { loadSb3, loadProjectJson, loadCloudId, ScratchProject, SpriteCollection, Sprite, BlockCollection, Block } from '../src/sb-util';
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
sorry for the huge diff on this file, but you didn't have prettier running on the tests folder, so I enabled it

Reply:
Friendly reminder this mode exists: 
![image](https://user-images.githubusercontent.com/549355/62672279-229d9200-b968-11e9-9fa5-a1303e46c2e4.png)